internal/traceprof/testapp/test_app.go

```diff
@@ -0,0 +1,8 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+// Package testapp has the protbuf/grpc definitions for the test application
+// implemented in traceproftest.
+package testapp
```

Comment:
> To avoid having the test packages testapp and teaceproftest somehow in the dependency graph, I suggest to prefix these with _ to have go tools ignore them.

Interesting idea! Just tried it, but I don't know how the go tool defines the word "ignored" — at least it doesn't seem like underscored pkgs are excluded from the dependency graph 🙈 

https://go.dev/play/p/Qx8SkS8ORO1

Assuming that it did work as expected, I think I could only do `_traceproftest` — `testapp` would have to stay as-is is order to be imported by the former.

> PS: Really cool you are adding those in the repo. Will use this as a source of inspiration for appsec.

Yeah, go for it! I'd also be happy to share the code if possible (might get too messy).